test/e2e/e2ethanos/services.go

```diff
@@ -856,7 +856,7 @@ func NewMinio(env e2e.Environment, name, bktName string) (*e2e.InstrumentedRunna
 			Command: e2e.NewCommandWithoutEntrypoint("sh", "-c", fmt.Sprintf(strings.Join(commands, " && "), minioKESGithubContent, minioKESGithubContent, bktName, 8090)),
 			//TODO(@clyang82): This is a temporary workaround for https://github.com/efficientgo/e2e/issues/9
 			//Readiness: e2e.NewHTTPReadinessProbe("http", "/minio/health/ready", 200, 200),
-			Readiness: e2e.NewCmdReadinessProbe(e2e.NewCommand("sh", "-c", "sleep 1 && curl -k https://127.0.0.1:8090/minio/health/ready")),
+			Readiness: e2e.NewCmdReadinessProbe(e2e.NewCommand("sh", "-c", "sleep 2 && curl -k https://127.0.0.1:8090/minio/health/ready")),
```

Comment:
Good catch. 
I think we can make this readiness probe use a loop retry with timeout. :)
```
curl --retry 5 --retry-delay 1
```